utils/helpers/Config.ts

```diff
@@ -0,0 +1,28 @@
+import { merge as _merge } from 'lodash';
```

Comment:
is it also possible without lodash?

Reply:
yes, but you would need to iterate all sub objects and all its types.